docs/source/stepfunctions.rst

```diff
@@ -0,0 +1,118 @@
+.. _stepfunctions:
+
+CfnCluster Stepfunctions
+########################
+
+Why Stepfunctions
+=================
+
+* Allows for complex workflows with CfnCluster
+* Handles cluster creation, teardown, and updates
+* Useful for conditional automated job execution
+* Has better interfacing with other AWS services
```

Comment:
Interfaces with other AWS services

Reply:
but now each line is not roughly the same character length :( fineeee